src/debugger/scriptImporter.ts

```diff
@@ -36,28 +37,37 @@ export class ScriptImporter {
             let waitForSourceMapping = Q<void>(null);
             if (sourceMappingUrl) {
                 /* handle source map - request it and store it locally */
-                waitForSourceMapping = this.writeSourceMap(sourceMappingUrl, scriptUrl)
+                waitForSourceMapping = this.writeAppSourceMap(sourceMappingUrl, scriptUrl)
                     .then(() => {
                         scriptBody = this.sourceMapUtil.updateScriptPaths(scriptBody, sourceMappingUrl);
                     });
             }
 
             return waitForSourceMapping
-                .then(() => this.writeScript(scriptBody, scriptUrl))
+                .then(() => this.writeAppScript(scriptBody, scriptUrl))
                 .then((scriptFilePath: string) => {
                     Log.logInternalMessage(LogLevel.Info, `Script ${scriptUrlString} downloaded to ${scriptFilePath}`);
                     return { contents: scriptBody, filepath: scriptFilePath };
                 }).finally(() => {
                     // Request that the debug adapter update breakpoints and sourcemaps now that we have written them
-                    return new Request().request(`http://localhost:${this.debugAdapterPort}/refreshBreakpoints`);
+                    return new Request().request(`http://localhost:${debugAdapterPort}/refreshBreakpoints`);
                 });
         });
     }
 
+    public downloadDebuggerWorker(sourcesStoragePath: string): Q.Promise<void> {
```

Comment:
I wouldn't put downloadDebuggerWorker here. ScriptImporter was cohesive before just being responsible for downloading the scripts that the RN App requests, and this method is not related to that responsibility.